deis-cleanup.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+require "open3"
+require "csv"
+
+KEEP_LAST_VERSIONS = ENV["KEEP_LAST_VERSIONS"].to_i || 2
```

Comment:
It will not work as you expecting. When there is no key in the `ENV`, `Hash#[]` will return a `nil` value. `nil.to_i` will give you `0`. You should either use `(ENV["KEY"] || 2).to_i` or `Hash#fetch` with default value (with `to_i`). Use the second one.